<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yeni_arac_giris</class>
 <widget class="QDialog" name="yeni_arac_giris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>457</width>
    <height>668</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>457</width>
    <height>668</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Yeni Araç</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#yeni_arac_giris{
image: url(:/yeniarac.png);
}</string>
  </property>
  <widget class="QLabel" name="aracformudurum">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#aracformudurum{
image: url(:/statusbar.png);
color: rgb(208, 0, 0);
font: 13pt &quot;MS Shell Dlg 2&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="yeni_arac_iptal">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>561</y>
     <width>100</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>321impact</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;İptal Et&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#yeni_arac_iptal{
color: rgb(255, 255, 255);
font: 11pt &quot;321impact&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(130, 0, 0, 255));
border-width: 1px;
border-color:#FEB201;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}
#yeni_arac_iptal:hover{
background-color: rgb(43, 72, 117);
border-color: rgb(130, 0, 0);
}
#yeni_arac_iptal:pressed{
background-color: rgb(0, 0, 127);
border-color: rgb(85, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>İptal Et</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="yeni_arac_yenile">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>561</y>
     <width>100</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>321impact</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Formu Tazele&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">#yeni_arac_yenile{
color: rgb(255, 255, 255);
font: 11pt &quot;321impact&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(130, 0, 0, 255));
border-width: 1px;
border-color:#FEB201;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}
#yeni_arac_yenile:hover{
background-color: rgb(43, 72, 117);
border-color: rgb(130, 0, 0);
}
#yeni_arac_yenile:pressed{
background-color: rgb(0, 0, 127);
border-color: rgb(85, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Yenile</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="yeni_arac_kaydet">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>561</y>
     <width>100</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>321impact</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Formu Kaydet&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#yeni_arac_kaydet{
color: rgb(255, 255, 255);
font: 11pt &quot;321impact&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(130, 0, 0, 255));
border-width: 1px;
border-color:#FEB201;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}
#yeni_arac_kaydet:hover{
background-color: rgb(43, 72, 117);
border-color: rgb(130, 0, 0);
}
#yeni_arac_kaydet:pressed{
background-color: rgb(0, 0, 127);
border-color: rgb(85, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="arac_renk">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>397</y>
     <width>188</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNoAutoUppercase|Qt::ImhPreferUppercase</set>
   </property>
  </widget>
  <widget class="QComboBox" name="arac_konumu">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>440</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="arac_marka_model">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>354</y>
     <width>188</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNoAutoUppercase|Qt::ImhPreferUppercase</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="arac_plaka_no">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>311</y>
     <width>188</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="arac_sure">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>477</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="musteri_adi">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>135</y>
     <width>188</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="musteri_cep_no">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>223</y>
     <width>188</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="musteri_soyadi">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>179</y>
     <width>188</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNoAutoUppercase|Qt::ImhPreferUppercase</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="musteri_tc_no">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>91</y>
     <width>188</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="SAdece Rakam Girmelisin">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;11 Haneli Tc. Kimlik Numarası&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>musteri_tc_no</tabstop>
  <tabstop>musteri_adi</tabstop>
  <tabstop>musteri_soyadi</tabstop>
  <tabstop>musteri_cep_no</tabstop>
  <tabstop>arac_plaka_no</tabstop>
  <tabstop>arac_marka_model</tabstop>
  <tabstop>arac_renk</tabstop>
  <tabstop>arac_konumu</tabstop>
  <tabstop>arac_sure</tabstop>
  <tabstop>yeni_arac_kaydet</tabstop>
  <tabstop>yeni_arac_yenile</tabstop>
  <tabstop>yeni_arac_iptal</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
